src/window.rs

```diff
@@ -1184,6 +1184,11 @@ impl Window {
     self.window.current_monitor()
   }
 
+  #[inline]
+  pub fn monitor_from_point(&self, x: f64, y: f64) -> Option<MonitorHandle> {
+    self.window.monitor_from_point(x, y)
+  }
+
```

Comment:
`monitor::from_point` main usage will be without a Window, so we also needs it to be exposed on `event_loop::EventLoopWindowTarget`

Reply:
sure

corresponding methods have added into this PR